apps/aecore/mix.exs

```diff
@@ -2,18 +2,24 @@ defmodule Aecore.Mixfile do
   use Mix.Project
 
   def project do
-    [
-      app: :aecore,
-      version: "0.1.0",
-      build_path: "../../_build",
-      config_path: "../../config/config.exs",
-      deps_path: "../../deps",
-      lockfile: "../../mix.lock",
-      elixir: "~> 1.5",
-      build_embedded: Mix.env() == :prod,
-      start_permanent: Mix.env() == :prod,
-      deps: deps()
-    ]
+    [app: :aecore,
+     version: "0.1.0",
+     build_path: "../../_build",
+     config_path: "../../config/config.exs",
+     deps_path: "../../deps",
+     lockfile: "../../mix.lock",
+     elixir: "~> 1.5",
+     compilers: [:make, :elixir, :app],
+     aliases: aliases,
```

Comment:
this will give a compiler warning without `()`

Reply:
Increasing the difficulty by hashing the solution was suggested by John Tromp.